htdocs/composer.json

```diff
@@ -23,7 +23,7 @@
     "leafo/scssphp": "v0.6.7",
     "league/commonmark": "^0.15.6",
     "sentry/sentry": "^1.7",
-    "opencaching/okapi": "dev-master as 1.1.x-dev"
+    "opencaching/okapi": "1.1.0"
```

Comment:
See https://github.com/opencaching/okapi/pull/514#issuecomment-333301905. I think the Okapi version number should be used here, or even better: Numer + 7-place commit ID.